docs/astrocut/index.rst

```diff
@@ -127,7 +127,7 @@ aligned and have the same pixel scale, no checking is done by Astrocut.
                 >>> center_coord = SkyCoord("150.0945 2.38681", unit='deg')
                 >>> cutout_size = [200,300]
                 
-                >>> png_files = img_cut(input_files, center_coord, cutout_size, img_format='png', drop_after="")    #doctest: +SKIP
+                >>> png_files = img_cut(input_files, center_coord, cutout_size, img_format='png')    #doctest: +SKIP
```

Comment:
This is a quick fix for a documentation issue I noticed while creating the different types of astrocut products.